a10_neutron_lbaas/v2/handler_listener.py

```diff
@@ -87,7 +117,9 @@ def _set(self, set_method, c, context, listener):
         persistence = handler_persist.PersistHandler(
             c, context, listener.default_pool)
 
+        # This doesn't do anything anymore.
         vport_meta = self.meta(listener.loadbalancer, 'vip_port', {})
+        # vport_meta.update(template_args)
```

Comment:
The comment refers to the line below, not my attempt at shoving listener data in.